### PR TITLE
Behat tests

### PR DESCRIPTION
**Pull request by Niktux**
Helper to run a full integration test.

make full-test:
 - build 2 containers (php cli with pecl amqp  rabbitmq)
 - configure rabbitmq (with a little trick : sleep 5s and hope that's enough for rabbitmq to boot)
 - launch behat tests
 - stop the containers

### CODE REVIEW

**Comment by Niktux** [comment]
Switch to Behat (wip - too technical features, hardcoded configuration)

```make behat``` and enjoy

**Comment by Niktux** [comment]
ping @lebris

File: behat.mk [hunk 1]
```diff
@@ -0,0 +1,69 @@
+RMQ_PORT=5672
```
Comment (lebris) [inline behat.mk]: `15672`

File: docker/images/apache/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+FROM php:7.1-cli
```
Comment (Niktux) [inline docker/images/apache/Dockerfile]: 5.6

**Comment by Niktux** [comment]
Ping @lebris to review and merge

**Pull request merged**